source/Halibut.Tests/BackwardsCompatibility/InvocationExceptionsAreMappedCorrectlyFixture.cs

```diff
@@ -10,28 +12,11 @@ namespace Halibut.Tests.BackwardsCompatibility
     public class InvocationExceptionsAreMappedCorrectlyFixture
     {
         [Test]
-        public async Task OldInvocationExceptionMessages_AreMappedTo_ServiceInvocationHalibutClientException_Polling()
+        [TestCaseSource(typeof(ServiceConnectionTypesToTest))]
+        public async Task OldInvocationExceptionMessages_AreMappedTo_ServiceInvocationHalibutClientException(ServiceConnectionType serviceConnectionType)
         {
             using (var clientAndService = await ClientAndPreviousServiceVersionBuilder
-                       .WithPollingService()
-                       .WithServiceVersion(PreviousServiceVersions.v5_0_429)
-                       .Build())
-            {
-                var echo = clientAndService.CreateClient<IEchoService>(se =>
-                {
-                    se.PollingRequestQueueTimeout = TimeSpan.FromSeconds(20);
```

Comment:
What happened to these timeouts?

Reply:
It's still there, it's a kind of nasty diff to read
![image](https://github.com/OctopusDeploy/Halibut/assets/86938706/e94d7ec6-d236-4eb1-bc00-0428e47c8ea5)

Reply:
2 tests become 1 by using the test case source